.appveyor.yml

```diff
@@ -59,6 +59,8 @@ install:
     if (!(Test-Path "$env:nupkg_cache_path")) { mkdir -Force "$env:nupkg_cache_path" }
     if (!(Test-Path "${env:nupkg_cache_path}\chocolatey.${chocoVersion}.nupkg")) { iwr "https://chocolatey.org/api/v2/package/chocolatey/${chocoVersion}" -OutFile   "${env:nupkg_cache_path}\chocolatey.${chocoVersion}.nupkg" }
     cup chocolatey --source ${env:nupkg_cache_path} --version $chocoVersion
+    if (!(Test-path "${env:nupkg_cache_path}\wormies-au-helpers.0.2.0.nupkg")) { iwr "https://chocolatey.org/api/v2/package/wormies-au-helpers/0.2.0" -OutFile "${env:nupkg_cache_path}\wormies-au-helpers.0.2.0.nupkg" }
```

Comment:
Is there a reason that you are pinning specifically to 0.2.0?  Would it be worth bringing this out as a variable, similar to chocoVersion?

Reply:
I'm pinning it due to caching, it's easier to cache distinct versions than if I catch everything.
It could be worth bringing it out as an environment variable, just didn't see the point when I added it.